src/compiler/utilitiesPublic.ts

```diff
@@ -761,6 +764,16 @@ export function unescapeLeadingUnderscores(identifier: __String): string {
 export function idText(identifierOrPrivateName: Identifier | PrivateIdentifier): string {
     return unescapeLeadingUnderscores(identifierOrPrivateName.escapedText);
 }
+
+/**
+ * If the text of an Identifier matches a keyword (including contextual and TypeScript-specific keywords), returns the
+ * SyntaxKind for the matching keyword.
+ */
+export function identifierToKeywordKind(node: Identifier) {

```

Comment:
Due to the rudimentary d.ts bundler, this one needs an explicit annotation:

```suggestion
export function identifierToKeywordKind(node: Identifier): KeywordSyntaxKind | undefined {
```